lib/api/interceptor.js

```diff
@@ -22,20 +22,21 @@ class XHRInterceptor {
 
       const mockedRequests = Scenarios.getCurrentMockedRequests();
       for (let mockedRequest of mockedRequests) {
-        const methodsMatch = request.method === mockedRequest.method;
-        const urlsMatch    = request.url === mockedRequest.url;
-        const paramsMatch  = request.body === mockedRequest.params;
-
-        if (methodsMatch && urlsMatch && paramsMatch && mockedRequest.active) {
-          return this.buildResponse(responder, mockedRequest.getResponse(mockedRequest.selectedStateId), request);
+        if (mockedRequest.matches({
```

Comment:
Ok so maybe
```
const {url, body, method} = xhrRequest;
if (mockedRequest.matches({url, params: body, method}) {

}
```

Just trying to make it a bit more readable the `})) {` part in line 29 looks very confusing to me.

We named it params in the request because body is a very general name and we want to map the xhr request body to our own model with a property name that is more obvious.